src/js/views/globals/prelogin/login-prompt.hbs

```diff
@@ -4,7 +4,7 @@
 
 <button class="prelogin__continue-button button--green js-login">
   Continue to sign in
-  <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="sign-in-alt" role="img" xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 512 512" class="svg-inline--fa fa-sign-in-alt">
+  <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="right-to-bracket" role="img" xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 512 512" class="svg-inline--fa fa-right-to-bracket">
```

Comment:
Coincides with the change made to the CSS selector change in the `src/js/views/globals/prelogin/prelogin.css` file.